components/button/index.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ButtonInput from './button';
+import { Story } from '@storybook/react';
+
+type ButtonProps = {
+  intext: string;
+  disabled: boolean;
+  onClick: () => void;
+};
+
+const Template: Story<ButtonProps> = (args) => <ButtonInput {...args} />;
+
+export const LogInButton = Template.bind({});
+LogInButton.args = {
+  intext: 'login',
+  disabled: false,
+};
+
+export const LogOutButton = Template.bind({});
+LogOutButton.args = {
+  intext: 'logout',
```

Comment:
What is intext? Should it be inText? How does localisation work here?

Reply:
@jjalan , sir not able to add localization in storybook, I tried and search about it how to add localization in storybook but not many resources available, tried few packages but they are not working.